bpf/init.sh

```diff
@@ -561,6 +561,17 @@ else
 	fi
 fi
 
+# Remove bpf_netdev.o from previously used devices
+for iface in $(ip -o -a l | awk '{print $2}' | cut -d: -f1 | grep -v cilium); do
+    [ "$iface" == "$NATIVE_DEV" ] && continue
+    if (tc filter show dev "$iface" ingress ; tc filter show dev "$iface" egress) | \
+		grep -q "bpf_netdev.o"; then
+
+        echo "Removing bpf_netdev.o from $iface"
+        tc qdisc del dev "$iface" clsact || true
```

Comment:
I'm a bit worried about a corner case where e.g. on ingress we'd have bpf_netdev but on egress some other attached prog and then we'd remove the whole qdisc, which may not be intentional. Wasn't there a case where we attach bpf_network to every device? /cc @joestringer @jrfastab